.coderabbit.yaml

```
@@ -0,0 +1,64 @@
+    - "!node_modules/**"
+    - "!.git/**"
+    - "!*.lock"
+    - "!assets/adaptive-icon.png"
+    - "!assets/icon.png"
+    - "!assets/splash-icon.png"
+    - "!assets/favicon.png"
+  
+  # Focus on these file types
+  include_patterns:
+    - "**/*.tsx"
+    - "**/*.ts" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)